index.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+var _ = require('lodash');
+
+var predicates = require('./predicates');
+
+/**
+ * A plugin for Irken applications that exposes keyboard shortcut functionality.
```

Comment:
this is specifically the plugin registration function.  don't use the module description to describe the top level function (or any APIs)